real_estate_module/models/estate_property.py

```diff
@@ -0,0 +1,215 @@
+        required=True,
+        help="Name of the property, e.g., Bhat House"
+    )
+
+    postcode = fields.Char(
+        string="Postcode",
+        help="Postal code of the property"
+    )
+
+    description = fields.Text(
+        string="Description",
+        help="Detailed description of the property"
+    )
+
+    date_availability = fields.Date(
+        string="Available Date",
+        copy=False,
+        default=fields.Date.today,
+        help="Date when the property will be available for sale"
+    )
+
+    expected_price = fields.Float(
+        string="Expected Price",
+        required=True,
+        help="Price at which the property is expected to be sold",
+        index="btree"
+    )
+
+    selling_price = fields.Float(
+        string="Selling Price",
+        readonly=True,
+        copy=False,
+        help="The price at which the property is actually sold"
+    )
+
+    bedrooms = fields.Integer(
+        string="No. of Bedrooms",
+        default=2,
+        help="Number of bedrooms in the property"
+    )
+
+    living_area = fields.Integer(
+        string="Living Area",
+        help="The total living area of the property in square meters"
+    )
+
+    facades = fields.Integer(
+        string="Facades",
+        help="Number of facades of the property"
+    )
+
+    garage = fields.Boolean(
+        string="Has Garage?",
+        help="Indicates whether the property has a garage"
+    )
+
+    garden = fields.Boolean(
+        string="Has Garden?",
+        help="Indicates whether the property has a garden"
+    )
+
+    garden_area = fields.Integer(
+        string="Garden Area",
+        help="The area of the garden in square meters"
+    )
+
+    garden_orientation = fields.Selection(
+        string="Garden Orientation",
+        selection=[
+            ('north', "North"),
+            ('south', "South"),
+            ('east', "East"),
+            ('west', "West")
+        ],
+        help="The orientation of the garden"
+    )
+
+    active = fields.Boolean(
+        string="Active",
+        default=True,
+        help="Indicates whether the property is active and available for sale"
+    )
+
+    state = fields.Selection(
+        string="Current State",
+        selection=[
+            ('new', "New"),
+            ('offer_received', "Offer Received"),
+            ('offer_accepted', "Offer Accepted"),
+            ('sold', 'Sold'),
+            ('cancelled', "Cancelled")
+        ],
+        required=True,
+        copy=False,
+        default='new',
+        help="The current state of the property"
+    )
+
+    property_type_id = fields.Many2one(
+        comodel_name='estate.property.type',
+        string="Property Type",
+        help="The type of the property (e.g., apartment, house, etc.)"
+    )
+
+    buyer = fields.Many2one(
+        comodel_name='res.partner',
+        string="Buyer",
+        help="The buyer who purchased the property"
+    )
+
+    salesman = fields.Many2one(
```

Comment:
```suggestion
    salesman_id = fields.Many2one(
```
Relational field `_id`